benches/coprocessor_executors/mod.rs

```diff
@@ -21,7 +21,7 @@ fn bench_table_scan_next(
 ) {
     use tikv::coprocessor::dag::executor::TableScanExecutor;
 
-    b.iter_with_setup(
+    b.iter_batched(
```

Comment:
Thanks a lot for modification. Please leave `coprocessor_executors` alone because we have a huge PR refining it: https://github.com/tikv/tikv/pull/4501